### PR TITLE
Missing accumulate tests

### PR DESCRIPTION
**Pull request by bramtayl**
[Uncovered](https://codecov.io/gh/JuliaLang/julia/src/master/base/accumulate.jl#L245)
[Uncovered](https://codecov.io/gh/JuliaLang/julia/src/master/base/accumulate.jl#L299)
[Uncovered](https://codecov.io/gh/JuliaLang/julia/src/master/base/accumulate.jl#L304)
[Uncovered](https://codecov.io/gh/JuliaLang/julia/src/master/base/accumulate.jl#L333)

### CODE REVIEW

**Review by dkarrasch — APPROVED** [review]

File: test/arrayops.jl [hunk 1]
```diff
@@ -2432,6 +2432,19 @@ end
         @test arr_cop ≈ cumprod(arr)
     end
 
+    # multidimensional, dims = 1 with init
+    @test accumulate(+, [1 1; 1 1], dims = 1, init = 1) == [2 2; 3 3]
+
+    @test_throws ArgumentError(
+        "acccumulate does not support the keyword arguments Symbol[:unsupported]"
```
Comment (dkarrasch) [inline test/arrayops.jl]: should be
```julia
"acccumulate does not support the keyword arguments [:unsupported]"
```
same below.

**Review by dkarrasch — CHANGES_REQUESTED** [review]

File: test/arrayops.jl (hunk 1 above)
Comment (bramtayl) [reply test/arrayops.jl]: Oops. Thanks!

**Review by dkarrasch — APPROVED** [review]
LGTM now.

**Comment by bramtayl** [comment]
Let me know if you want me rebase this

**Pull request closed without merging**